### DISCUSSION TITLE
Downloading songs/IP blocked warning

### DISCUSSION THREAD

**Discussion by Bwangster12** [KRTirtho/spotube#677] [Q&A]
I've used a few comparable apps before, but this is the first time I've ever thought about my IP being blocked on YouTube for downloading songs.  Is this a real concern?  I am signed in to my primary YouTube account, and don't download a lot....but if I download 100 or so songs from time to time for a trip, is there a real concern?  Is there a risk to my Google account getting banned or is it just a matter of changing my IP?

**Comment by KRTirtho** [comment]
Yes, it is concerning if you're using the official YouTube API (u change that from "Settings" > "API Type"). If you're using Piped API there's no worries

> if I download 100 or so songs from time to time for a trip, is there a real concern?

I'm not sure of the exact rate limit but yes downloading 100 songs at once can get your IP blocked

>  Is there a risk to my Google account getting banned or is it just a matter of changing my IP?

Your account won't get banned. Only your IP will be blocked for couple of months. That means no matter what account you use in that network, certain videos might not load (especially the ones you're tried to download)
And yes you can change your IP just by asking the ISP. But I'd suggest using a VPN to avoid this kind risk

**Reply by SleuthChromiumUser** [reply]
Thank you
